tagging/hierarchy/get_manifests.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+from tagging.hierarchy.images_hierarchy import ALL_IMAGES
+from tagging.manifests.manifest_interface import ManifestInterface
+
+
+def get_manifests(image: str | None) -> list[ManifestInterface]:
```

Comment:
After this refactoring, there is no need to have both taggers and manifests in the same place.
So, it's better to have 2 functions.
Their implementation is similar, but that's ok